jupyter_config.json

```diff
@@ -1,4 +1,7 @@
 {
+  "Session": {
+    "session": "eb4dcf6e-f1f5-422e-b7b0-cdcf10355ace"
```

Comment:
I'm not 100% sure this part is needed but my thinking is that this will make sure that all instances of this application share the same session id.